client/checkout/classic/upe-split.js

```diff
@@ -539,6 +565,11 @@ jQuery( function ( $ ) {
 					},
 				},
 			};
+			if ( PAYMENT_METHOD_IDS.AFTERPAY_CLEARPAY === paymentMethodType ) {
```

Comment:
I have to yet add proper comment explaining this line

Reply:
I was about to request this.